views/alpha.pug

```diff
@@ -30,6 +30,7 @@ block content
                 .input-group-btn
                     button.btn.btn-default.btn-lg(type='button', data-el=name)
                         span.caret
+            span.help-block Click to copy
```

Comment:
I realized that there seems to be a lot of redundancy here with 

```
         .form-group
             label(for=id) Complete CSS
             .input-group.input-group-lg
                 input.form-control(id=id, type='text', value=file)
                  .input-group-btn
                      button.btn.btn-default.btn-lg(type='button', data-el=name)
                          span.caret
             span.help-block Click to copy
```

I realize that this isn't due to this PR, but am thinking we could abstract the whole section to an include. Perhaps in another PR -- which I'm happy to submit once this is merged.